agent-tooling/agent-tooling.gradle

```diff
@@ -3,7 +3,8 @@ apply from: "${rootDir}/gradle/java.gradle"
 minimumBranchCoverage = 0.6
 minimumInstructionCoverage = 0.8
 excludedClassesCoverage += [
-  'io.opentelemetry.auto.tooling.*'
+  'io.opentelemetry.auto.tooling.*',
```

Comment:
What is the purpose of excluding auto tooling classes from test coverage in their own auto tooling module? What coverage will this module verify?

Reply:
The purpose is to make the build pass 🤣. You're right, this module doesn't verify any coverage.

Reply:
But what is the point of "bringing back code coverage verification" if we exclude the whole module from verification? Wouldn't it be better to lower minimum coverage requirement instead? Currently it may appear like this module satisfies coverage expectation, judging by  `minimumInstructionCoverage = 0.8`, when in reality it may be close to 0.

Reply:
Do you think it makes more sense here to set `minimumInstructionCoverage = 0` and remove the exclusions?

Reply:
I think so. It clearly communicates that current expected coverage for this module is 0. And does not create false sense of safety, which "0.8" kinda does :)

Reply:
👍 submitted this change in #365